lms/djangoapps/organizations/views.py

```diff
@@ -175,3 +184,27 @@ def groups(self, request, pk):
             organization.groups.add(group)
             organization.save()
             return Response({}, status=status.HTTP_201_CREATED)
+
+    @action(methods=['get', ])
+    def user_ids(self, request, pk):
```

Comment:
@mattdrayer we already have two endpoints for users and groups whereas we don't need workgroup ids for org. I have modified existing `users` and `groups` endpoints by adding a `view` parameter to get ids like this `view=ids`.